recipe-client-addon/lib/FilterExpressions.jsm

```diff
@@ -41,3 +43,32 @@ this.FilterExpressions = {
     return jexl.eval(onelineExpr, context);
   },
 };
+
+/**
+ * Return an array of the given object's own keys (specifically, its enumerable
+ * properties), or undefined if the argument isn't an object.
```

Comment:
FWIW, I can see arguments both ways and don't really mind either way, so approved the PR. I'm also painfully aware that small changes like this sometimes get more scrutiny than much bigger ones, because humans, and that's not necessarily justified - so carry on! (see also: the now possibly-slightly-cult-ified bikeshed v. nuclear reactor example from Parkinson's law of triviality)